src/main/java/Controller/playSongController.java

```diff
@@ -0,0 +1,25 @@
+package Controller;
+
+import Entities.Song;
+import InputBoundary.playSongInputBoundary;
+import InputData.songInputData;
+
+/**
+ * Controller for the playSong use case
+ */
+public class playSongController {
+
+    private final playSongInputBoundary playSongInteractor;
+    private final songInputData songInputData;
+
+    public playSongController(playSongInputBoundary playSongInteractor, Song song) {
+        this.playSongInteractor = playSongInteractor;
```

Comment:
After the conversation we had in tutorial today, I was under the impression that the controller will be _creating_ an instance of the interactor, as opposed to being passed one.